### DISCUSSION TITLE
Feature Request: Support for Disabling p-stepperPanel in p-stepper

### DISCUSSION THREAD

**Discussion by otaldopira** [primefaces/community#2428] [PrimeNG]
Description
I would like to request a new feature for the p-stepper component in PrimeNG. Currently, there is no support for dynamically disabling or removing p-stepperPanel.

Use Case
In my application, I need a dynamic p-stepper where the number of steps can change based on user inputs. Depending on the responses given, steps (i.e., p-stepperPanel elements) should be added or removed.

I attempted to use conditional rendering with *ngIf, but this approach does not work as expected. The p-stepperPanel elements are rendered based on the initial condition, and when the value changes, the stepper does not properly update to reflect the addition or removal of steps.

**Comment by bqotm** [comment]
I second this !
just wondering why this hasn't been thought of earlier

**Comment by KeemOnGithub** [comment]
I was able to disable/enable the entire p-stepper by setting `pointer-events: none` to p-stepper. Using Angular, it can be done dynamically by adding this to p-stepper:

`[style.pointer-events]="disableStepper ? 'none' : 'auto'"`

**Reply by eric-denkievicz** [reply]
Thanks for the contribution, but for my case, I need to remove it from the user's view.

**Reply by TheYouIsHere** [reply]
@eric-denkievicz Let me know if you managed to solve this!

**Reply by eric-denkievicz** [reply]
Sorry @TheYouIsHere , but I haven't been able to resolve this yet.
